### ISSUE TITLE
Stats bar chart should announce data with a screen reader

### ISSUE THREAD

**Issue by mcbouslog** [zooniverse/front-end-monorepo#5842]
## Package
_Choose from the list:_
- [x] lib-user

## Is your feature request related to a problem? Please describe.
The lib-user BarChart component, used for user stats and group stats, should announce the individual bar data with a screen reader. Currently it does not, possibly due to a Grommet bug with the Details component within the DataChart component missing a11yTitles/aria-labels.

## Describe the solution you'd like
The BarChart component announces data similar to [the YourStats component](https://zooniverse.github.io/front-end-monorepo/@zooniverse/fe-project/index.html?path=/story/project-app-screens-classify-your-stats--your-stats) (built with visx).

**Comment by mcbouslog** [comment]
Closed with [grommet PR 7310](https://www.github.com/grommet/grommet/pull/7310)

**Comment by mcbouslog** [comment]
Closed with [grommet PR 7310](https://www.github.com/grommet/grommet/pull/7310)
